tests/qos/qos_sai_base.py

```diff
@@ -8,12 +9,79 @@
 from tests.common.helpers.assertions import pytest_assert, pytest_require
 from tests.common.mellanox_data import is_mellanox_device as isMellanoxDevice
 from tests.common.dualtor.dual_tor_utils import upper_tor_host,lower_tor_host,dualtor_ports
+from tests.common.dualtor.mux_simulator_control import get_mux_status
 from tests.common.dualtor.mux_simulator_control import mux_server_url, toggle_all_simulator_ports
 from tests.common.dualtor.constants import UPPER_TOR, LOWER_TOR
 from tests.common.utilities import check_qos_db_fv_reference_with_table
+from tests.common.utilities import wait_until
 
 logger = logging.getLogger(__name__)
 
+def check_mux_status(duthosts, active_side):
```

Comment:
@XuChen-MSFT this function looks nice, and pretty common for dual tor, could you check if any available function already? If not, could we move this function to tests/common/dualtor/dual_tor_utils.py?

Reply:
"check_mux_status()" and "validate_check_result" have been implemented in "tests\dualtor\test_toggle_mux.py".

To avoid duplicated code, consider that moving these functions to "tests\common\dualtor\mux_simulator_control.py" as common utils for mux cable.

@StormLiangMS  @lolyu   any comments for above idea?